azext_edge/edge/commands_edge.py

```diff
@@ -470,14 +470,20 @@ def migrate_assets(
     resource_group_name: str,
     name_patterns: Optional[list[str]] = None,
```

Comment:
saw the related flag `--name-pattern` changed to `--pattern`, should we update this variable name as well?